routers/routes/routes.go

```diff
@@ -651,7 +651,7 @@ func RegisterRoutes(m *macaron.Macaron) {
 		if setting.HasRobotsTxt {
 			ctx.ServeFileContent(path.Join(setting.CustomPath, "robots.txt"))
 		} else {
-			ctx.Error(404)
+			ctx.Status(404)
```

Comment:
Sure? As I looked at other locations in the code where it returns a working 404.